src/Parser/EnhancementShaman/Modules/ShamanCore/GCD.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Icon from 'common/Icon';
+import { formatPercentage } from 'common/format';
+
+import Module from 'Parser/Core/Module';
+
+import StatisticBox, { STATISTIC_ORDER } from 'Main/StatisticBox';
+
+class GCD extends Module {
+  suggestions(when) {
+    const deadTimePercentage = this.owner.modules.alwaysBeCasting.totalTimeWasted / this.owner.fightDuration;
+    when(deadTimePercentage).isGreaterThan(0.2)
+      .addSuggestion((suggest, actual, recommended) => {
+        return suggest(`Your dead GCD time can be improved. Try to Always Be Casting (ABC); cast instant spells during movement phases and focus on having no delays between spell casts`)
+          .icon("spell_mage_altertime")
+          .actual(`${Math.round(deadTimePercentage * 100)}% dead GCD time`)
+          .recommended(`<${(formatPercentage(recommended))}% is recommended`)
+          .regular(recommended).major(recommended + 0.35);
+      });
+  }
+
+  statistic() {
```

Comment:
This statistic would be better placed in the Always Be Casting module where it's also calculated.